### PR TITLE
Aurn Script Reorganising

### PR DESCRIPTION
**Pull request by douglowe**
I've split out the different functions in the AURN postprocessing_organisation routine - so that we have two parallel routines, one for imputing data, the other for simply organising the data without imputing it, followed by a single routine which calculates the daily mean / max values, and returns the final dataframe for outputting. This is to bring it inline with the met processing scripts.

There's also some small changes to the datetime function loading (again to bring consistency across the aurn and met scripts).

### CODE REVIEW

**Review by AnnAnnFryingPan — COMMENTED** [review]
Please add a parameter description to the define_years_to_download function (for years)

**Review by AnnAnnFryingPan — COMMENTED** [review]
In doc-strings, don't use words like 'your', keep it objective.  E.g. in define_years_to_download, years is a parameter to that function rather than belonging to a reader. 

Also "change it to do this" is not very clear at all. Change what and how?
Also, how will 'Creat[ing] two new columns of datetimes for earliest and latest solve this?

**Review by AnnAnnFryingPan — COMMENTED** [review]
Looks good on the whole.

**Comment by douglowe** [comment]
> Please add a parameter description to the define_years_to_download function (for years)

I've realised I was just passing self.years - so have removed the parameter from the function call, and am loading it directly in the function.

File: environmental_data_modules/aurn_extractor.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def define_years_to_download(self, subset_df, years):
         datetime_start = pd.to_datetime(subset_df['start_date'].values, format='%Y/%m/%d').year
         # Problem with the end date is it could be ongoing. In which case, convert that entry into a date
```
Comment (AnnAnnFryingPan) [inline environmental_data_modules/aurn_extractor.py]: Lines 183 above, use of 'your'. Do a search on 'Your' and (e.g.) 'our' and replace with objective description of variable names.

Comment (douglowe) [reply environmental_data_modules/aurn_extractor.py]: I have reviewed the comments in the code - they were rather unhelpful! I've made them more relevant to what the code is doing (and more objective).

**Pull request merged**